resources/c/partial/tab.sass

```diff
@@ -0,0 +1,38 @@
+
+div.tab
+  float: left
+  position: absolute
+  right: 33vw
+  width: 10%
+  margin-top: 1rem
+  padding-bottom: 1rem
+
+div.tab button
+  display: block
+  color: white
+  width: 100%
+  height: 50%
+  outline: none
+  text-align: center
+  cursor: pointer
+  transition: 0.3s
+  font-family: Vanadine
+  background-color: #3f1ca2
+  border: none
+
+
+div.tab button span
+  display: inline-block
+  transform: rotate(-90deg)
+  width: 50px
+
+.inactive
+  opacity: .6
+
+
+.tabcontent
+  float: left
+  padding: 0px 12px
+  width: 70%
+  border-left: none
+  height: 300px
```

Comment:
I'm not sure how you're getting this stylesheet file to apply... It needs to be imported in `application.sass` to take effect.